README.md

```diff
@@ -1,37 +1,38 @@
 # Ember Octane vs Classic Cheat Sheet
 
-Check out [the deployed app](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)!
+Check out [the deployed app](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/). Yep, it uses Ember Octane!
 
-See side-by-side examples of Classic Ember vs [Octane Edition](https://emberjs.com/editions).
+You can see side-by-side examples of [Ember Octane](https://emberjs.com/editions/octane) and Ember Classic.
 
-This uses [Prism](https://prismjs.com/) code hightlighting with the [file highlight](https://prismjs.com/plugins/file-highlight/) plugin.
 
-Inspired by [YouMightNotNeedJQuery](http://youmightnotneedjquery.com/).
+## Running Locally
```

Comment:
Should this also be in `CONTRIBUTING.md`? Seems like the only reason to run locally would be to contribute.

Reply:
Yep, I think it may be a good idea. Will move the text to contributing.